frontend/src/pages/IstioConfigNew/IstioConfigNewPage.tsx

```diff
@@ -202,24 +226,44 @@ class IstioConfigNewPageComponent extends React.Component<Props, State> {
   };
 
   onIstioResourceCreate = () => {
+    if (this.props.activeClusters.length > 0) {
+      this.props.activeClusters.forEach(cluster => {
+        this.onIstioResourceCreateForCluster(cluster.name);
+      });
+    } else {
+      this.onIstioResourceCreateForCluster();
+    }
+  };
+
+  onIstioResourceCreateForCluster = (cluster?: string) => {
     const jsonIstioObjects: { namespace: string; json: string }[] = this.state.itemsPreview.map(item => ({
       namespace: item.items[0].metadata.namespace || '',
       json: JSON.stringify(item.items[0])
     }));
 
     this.promises
```

Comment:
What about something like this? So at least, in case of any failure, the user receives a success message for the Istio configs that where created successfully: 
```
    let errors = 0
    await Promise.all(jsonIstioObjects.map(o => API.createIstioConfigDetail(o.namespace, DIC[this.props.objectType], o.json, cluster))
    .map(p => p.catch(error => {
      AlertUtils.addError(
        'Could not create Istio ' +
        this.props.objectType +
        ' objects' +
        (cluster ? ' in cluster ' + cluster + '.' : '.'),
        error
      )
      errors++
    }))).then(() => {
      AlertUtils.add(
        'Istio ' + this.props.objectType + ' created' + (cluster ? ' in cluster ' + cluster : ''),
        'default',
        MessageType.SUCCESS
      );
    });

    if (errors == 0) {
      this.backToList();
    }
```

![image](https://github.com/kiali/kiali/assets/49480155/6154718a-7f51-44b0-b2e3-407b8e76aeb8)

Reply:
Thank you Josune for the hints, will try them.